db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is defined as a `CharField`. If the bonus is meant to represent a numeric value, it might be more appropriate to use a `DecimalField` or `IntegerField` depending on the requirements.